src/index.html

```diff
@@ -43,6 +43,7 @@
               <li data-ng-class="{active: $state.includes('news')}"><a data-ui-sref="news.list">News</a></li>
               <li data-ng-show="canManagePermissions" data-ng-class="{active: $state.includes('admin.permissions')}"><a data-ui-sref="admin">Permissions</a></li>
               <li data-ng-class="{active: $state.includes('about')}"><a data-ui-sref="about">About</a></li>
+              <li><a href="/wiki">Wiki</a></li>
```

Comment:
The stylization is done. However the actual linked content should stay, due to how we are going to be setting up the wiki in the future.